presentation/src/main/java/com/architect/coders/mu8/events/EventsViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package com.architect.coders.mu8.events
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.architect.coders.mu8.utils.ScopedViewModel
+import com.architect.codes.mu8.events.EventsUseCase
+import kotlinx.coroutines.launch
+
+class EventsViewModel(private val eventsUseCase: EventsUseCase) : ScopedViewModel() {
+
+    private val _model = MutableLiveData<EventsUiModel>()
+    val model: LiveData<EventsUiModel>
+    get() { if (_model.value == null) getEvents()
```

Comment:
This is missing the right format. Please fix it to improve its reading.

Reply:
Done